CHANGELOG.md

```diff
@@ -1,77 +1,77 @@
+- Added `AUTOMATION VIEW` for Audio Clips.
+- Added a menu for song parameters, accessible in Song View and Arranger View by pressing on the Select Encoder.
+- Added a `AFFECT ENTIRE GLOBAL FX MENU` to Kits, accessible in Kit Clip View by pressing on the Select Encoder with Affect Entire enabled.
+- Added support for LUMI Keys SYSEX protocol protocol. When hosting a LUMI Keys keyboard, the current scale will automatically be set on the keyboard.
+- Short-press of empty clip pads in Grid mode while REC is enabled automatically creates a new clip and queues it for recording at the start of the next bar.
+- Fixed a bug preventing clip selection while Shift was held.
+- Fixed numerous bugs, including some crash bugs, around the display of quantized stutter.
+- Fixed a bug with shift+scroll on small menus which would allow moving off the end of the menu, causing crashes.
+- Fixed several bugs with pad grid rendering.
+
+In addition, a number of improvements have been made to how the OLED display is used:
+
+- Added parameter name to Mod (Gold) Encoder popups.
+- "ARRANGER VIEW" and "SONG VIEW" now display the name of the current view on the screen.
+- The 12TET note name is now displayed along with the MIDI note number.
+- Added a new community setting which allows emulating the old 7SEG style on the OLED display.
+- Fixed several cases where popups could get stuck open.
+- Fixed a number of minor rendering bugs.
+
+### Sequencer
+
+- When changing instrument presets in session or arranger view all presets are now changed. Individual clips can still have their presets changed by first entering the clip.
+- Added support for 5 and 6 note scales.
+- Added 8 new built-in scales: Melodic Minor, Hungarian Minor, Marva (Indian), Arabian, Whole Tone, Blues, Pentatonic Minor, Hirajoshi.
+- Added "play `ONCE`" clip launch mode, settable by turning the Select Encoder left while holding the audition pad in Song view. Causes the clip to play once when triggered and then mute without changing the section.
+- Added `NOT FILL` note probability. Similar to the `FILL` probability but only plays when the `FILL` button is *not* pressed.
+- Added support for copy/paste of single rows.
+- Added support for "gentle paste" of notes which pastes notes without removing old ones.
+- Fixed numerous crash bugs around parameter automation when entering and leaving clip view.
+
+### Audio Clips
+- Changed default ModFX type is now DISABLED rather than FLANGER.
+
+### Kits
+
+- Drum randomization is no longer limited to only 10 sounds per folder.
+- Fixed several crashes related to drum randomization.
+
+### MIDI
+
+- Added `MIDI FOLLOW MODE` which causes MIDI data to be directed to the currently focused clip.
+- Added MIDI feedback, so external MIDI controllers can be made aware of the state of the Deluge synth engine. Configurable via the new global `MIDI > MIDI-FOLLOW > FEEDBACK` menu.
+- Added Loopy Pro and TouchOsc templates for MIDI feedback.
```

Comment:
These templates not just for MIDI feedback. They are for MIDI Follow and MIDI Feedback.